[piny i cleanup]

Bart3kTK opened the pull request:
ustawilem piny zgodnie ze schematem
(sprawdz mnie ktos dla pewnosci xd)

i pominalem piny 13, 14, 21, 37, 38, 39, 40, 41, 42, 47, 48 bo nie wiem jeszcze do czego to sluzy ale niedlugo sie dowiem

no i zrobilem maly cleanup do tych zmian

Commit:
[piny i cleanup]

ustawilem piny zgodnie ze schematem
(sprawdz mnie ktos dla pewnosci xd)

i pominalem piny 13, 14, 21, 37, 38, 39, 40, 41, 42, 47, 48
bo nie wiem jeszcze do czego to sluzy ale niedlugo sie dowiem

no i zrobilem maly cleanup do tych zmian

AlaiveCYA reviewed (commented):
Na tel nie widze dokładnie, napewno wszystkie define'y są w Kconfig?🍄

AlaiveCYA commented on components/mcu_cfg/mcu_gpio_config.c:
arm igniterów może ci się przydać

AlaiveCYA commented on components/mcu_cfg/mcu_gpio_config.h:
to samo z armem

AlaiveCYA commented on sdkconfig:
wrzuć ten plik do .gitignore

AlaiveCYA reviewed (changes requested):
💅

Commit:
[piny i cleanup]
    *popraweczka*

    ustawilem piny zgodnie ze schematem
    (sprawdz mnie ktos dla pewnosci xd)

    i pominalem piny 13, 37, 38, 39, 40, 41, 42, 47, 48
    bo nie wiem jeszcze do czego to sluzy ale niedlugo sie dowiem

    ---- poprawka -----
    dalem ten sdk do gitignore
    i okazalo sie ze nie mialem jeszcze valve_control to przekopiowalem
    z tanwa com

Commit:
ignore sdkconfig

Commit:
[szybka poprawka]
*szybkosc*
